releasenotes/notes/vm-healthchecking.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+releaseNotes:
+  - |
+    **Added** Configuration-Based VM Healthchecking
```

Comment:
skip release notes for now. it becomes messy. workload group conveys all the info

Reply:
sure